popSimManu.tex

```diff
@@ -252,6 +270,7 @@ \section*{Introduction}
 \label{fig:cartoon}
 \end{center}
 \end{figure}
+% ALG: I don’t see default generation time and mutation rate in the fig 
```

Comment:
true - but that'd be hard to fit in there? I'm OK with it being omitted.

Reply:
Looks good without it.